### ISSUE TITLE
Add toString to GeoJSON models

### ISSUE THREAD

**Issue by tobrun** [mapbox/mapbox-java#499]
Add `Object#toString` implementation to `com.mapbox.services.commons.geojson` classes

Current output:
```
com.mapbox.services.commons.geojson.Feature@5cd82fe
```

Expected output:
```
{
  "type": "Feature",
  "geometry": {
    "type": "Point",
    "coordinates": [125.6, 10.1]
  },
  "properties": {
    "name": "Dinagat Islands"
  }
}
```

**Comment by cammace** [comment]
Work for adding all toString methods has been done in https://github.com/mapbox/mapbox-java/pull/450 but is held up since we'd like to do something a bit more automated (like using AutoValue). It would be nice to get that decision made soon so we can move forward. For GeoJson the `toString` method could simply make use of the `toJson` method which creates a string just like the above expected output.

@zugaldia how would you like to move forward on this?

**Comment by zugaldia** [comment]
> @zugaldia how would you like to move forward on this?

The AutoValue approach from https://github.com/mapbox/mapbox-java/pull/491 (which I prefer) is unfortunately a breaking API change. How about we just merge https://github.com/mapbox/mapbox-java/pull/450 for now?

**Comment by cammace** [comment]
Resolved in 3.0, you can either use `toJson` or `toString` which will result in a json string or the typical java object information respectively.
